src/app/entities/course.ts

```diff
@@ -6,7 +6,7 @@ interface CourseItem {
 	duration: number;
 }
 
-class Course implements CourseItem {
+export class Course implements CourseItem {
```

Comment:
actually it was like that in the bolerplate so I decided to change it to see what fits better for me :)